packages/plugin-jsdocs/README.md

```diff
@@ -24,28 +24,28 @@ Measured documentation types are mapped to Code PushUp audits in the following w
 2. Install as a dev dependency with your package manager:
 
    ```sh
-   npm install --save-dev @code-pushup/doc-coverage-plugin
+   npm install --save-dev @code-pushup/jsdocs-plugin
    ```
 
    ```sh
-   yarn add --dev @code-pushup/doc-coverage-plugin
+   yarn add --dev @code-pushup/jsdocs-plugin
    ```
 
    ```sh
-   pnpm add --save-dev @code-pushup/doc-coverage-plugin
+   pnpm add --save-dev @code-pushup/jsdocs-plugin
    ```
 
 3. Add this plugin to the `plugins` array in your Code PushUp CLI config file (e.g. `code-pushup.config.ts`).
 
    ```js
-   import docCoveragePlugin from '@code-pushup/doc-coverage-plugin';
+   import jsDocsPlugin from '@code-pushup/jsdocs-plugin';
 
    export default {
      // ...
      plugins: [
        // ...
-       docCoveragePlugin({
-         sourceGlob: ['**/*.ts'],
+       jsDocsPlugin({
+         patterns: ['**/*.ts'],
        }),
```

Comment:
In the interest of keeping the first usage example as minimal as possible, I would use the shorthand here.

```suggestion
       jsDocsPlugin(['**/*.ts']),
```